api/incidents/incidentsRouter.js

```diff
@@ -429,6 +429,9 @@ router.get('/download', async (req, res) => {
     const state = req.query.state || null;
     let start = req.query.start || null;
     let end = req.query.end || null;
+    let selectedIncidents = req.body.selectedIncidents || null;
+    let selectedTags = req.body.selecteTags || null;
+    console.log(req.body);
```

Comment:
please remove all `console.logs()`  this is actually a potential security risk as well. we shouldn't print the req.body in production. It's really good for debugging in development, but shouldn't be left in production.